src/resources/postcss/views/month/_multiday-event.pcss

```
@@ -14,51 +14,84 @@
 	}
 
 	.tribe-events-calendar-month__multiday-event {
-		background-color: var(--color-background);
-		overflow: hidden;
-		position: relative;
+		height: 100%;
 		width: 100%;
-		z-index: var(--z-index-multiday-event);
+
+		.tribe-events-calendar-month__multiday-event-bar {
+			background-color: var(--color-background);
+			overflow: hidden;
+			position: relative;
+			width: 100%;
+			z-index: var(--z-index-multiday-event-bar);
+		}
 	}
 
+	.tribe-events-calendar-month__multiday-event {}
+
 	.tribe-events-calendar-month__multiday-event--width-2 {
-		width: calc(200% + 1px);
+		.tribe-events-calendar-month__multiday-event-bar {
+			width: calc(200% + 1px);
+		}
 	}
 
 	.tribe-events-calendar-month__multiday-event--width-3 {
-		width: calc(300% + 2px);
+		.tribe-events-calendar-month__multiday-event-bar {
+			width: calc(300% + 2px);
+		}
 	}
 
 	.tribe-events-calendar-month__multiday-event--width-4 {
-		width: calc(400% + 3px);
+		.tribe-events-calendar-month__multiday-event-bar {
+			width: calc(400% + 3px);
+		}
 	}
 
 	.tribe-events-calendar-month__multiday-event--width-5 {
-		width: calc(500% + 4px);
+		.tribe-events-calendar-month__multiday-event-bar {
+			width: calc(500% + 4px);
+		}
 	}
 
 	.tribe-events-calendar-month__multiday-event--width-6 {
-		width: calc(600% + 5px);
+		.tribe-events-calendar-month__multiday-event-bar {
+			width: calc(600% + 5px);
+		}
 	}
 
 	.tribe-events-calendar-month__multiday-event--width-7 {
-		width: calc(700% + 6px);
+		.tribe-events-calendar-month__multiday-event-bar {
```

```suggestion

		.tribe-events-calendar-month__multiday-event-bar {
```
This applies to all the `--width` modifiers above